Header header

uint32 seq
uint64 time

geometry_msgs/Vector3 orientation

# Standard deviation of orientation in yaw, pitch and roll
geometry_msgs/Vector3 stddev_orientation

float32 depth
float32 speed_of_sound
float32 salinity
float32 temperature
float32 pressure
float32 pressure_variance

uint8[8] adc_channels

uint64 min_preping_wait

uint16 self_test_result
uint32 status_word
